src/python/zquantum/core/cost_function_test.py

```diff
@@ -1,10 +1,84 @@
 import numpy as np
 import pytest
 from unittest import mock
-from .cost_function import AnsatzBasedCostFunction
+from sympy import Symbol
+from .cost_function import AnsatzBasedCostFunction, get_ground_state_cost_function
 from .interfaces.mock_objects import MockQuantumSimulator, MockEstimator, MockAnsatz
 from openfermion import QubitOperator
 
+RNGSEED = 1234
+
+
+@pytest.fixture
+def ground_state_cost_function():
```

Comment:
I think it would be more scalable if you passed all (or at least some) the params as fixture params.

Reply:
Addressed here: 5c13656